cltk/corpus/utils/importer.py

```diff
@@ -26,7 +26,7 @@
 from cltk.corpus.french.corpora import FRENCH_CORPORA
 from cltk.corpus.marathi.corpora import MARATHI_CORPORA
 from cltk.corpus.gujarati.corpora import GUJARATI_CORPORA
-from cltk.corpus.Medieval_texts.corpora import Medieval_CORPORA
+from cltk.corpus.Medieval.corpora import Medieval_CORPORA
```

Comment:
Sorry, but `Medieval` is not a valid namespace for us. This must be a language name and be in lowercase.